docs/release-notes/NuGet-6.4.md

```diff
@@ -0,0 +1,109 @@
+---
+title: NuGet 6.4 Release Notes
+description: Release notes for NuGet 6.4 including new features, bug fixes, and DCRs.
+author: jebriede
+ms.author: jebriede
+ms.date: 10/27/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.4 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.4**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.4](https://visualstudio.microsoft.com/downloads/) | [7.0.100](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 17.4 with .NET Core workload
+
+## Summary: What's New in 6.4
+
+* [DCR]: Package Source Mapping API does not support saving  - [#11935](https://github.com/NuGet/Home/issues/11935)
+
+* [Feature]: Add ability to designate a package reference as used by every project in the repo - [#10159](https://github.com/NuGet/Home/issues/10159)
+
+### Issues fixed in this release
+
+**DCRs:**
+
+* [DCR]: Static graph-based restore should handle an AggregateException from MSBuild - [#12100](https://github.com/NuGet/Home/issues/12100)
+
+* [DCR]:  Disable signed package verification by default on Linux in .NET 7 SDK - [#12043](https://github.com/NuGet/Home/issues/12043)
+
+* Signing:  use separate fallback certificate bundles for code signing and timestamping - [#12033](https://github.com/NuGet/Home/issues/12033)
+
+* Signing:  enable signed package verification by default on Linux in .NET 7 SDK - [#11973](https://github.com/NuGet/Home/issues/11973)
+
+* [DCR]: Central package management package source mapping should only look at configured feeds - [#11951](https://github.com/NuGet/Home/issues/11951)
+
+* [DCR]: Remove preview message when using central package management - [#11950](https://github.com/NuGet/Home/issues/11950)
```

Comment:
@jeffkl, @JonDouglas wyt about adding this in the what's new question.

Something along the lines of "Syntax complete", that's the buzzword I have used.

Reply:
Whatever words Jeff wants. "Production ready" sounds down that alley.

Reply:
cc @jeffkl 

Any suggestions?

Reply:
I would put this:

> Central package management is considered production ready and the preview message has been removed.